test/sagemaker_tests/tensorflow/tensorflow2_training/integration/sagemaker/test_experiments.py

```diff
@@ -62,14 +62,13 @@ def test_training(sagemaker_session, ecr_image, instance_type, framework_version
         resource_path = os.path.join(os.path.dirname(__file__), "..", "..", "resources")
         script = os.path.join(resource_path, "mnist", "mnist.py")
         estimator = TensorFlow(
+            model_dir=False,
             entry_point=script,
             role="SageMakerRole",
-            train_instance_type=instance_type,
-            train_instance_count=1,
+            instance_type=instance_type,
+            instance_count=1,
             sagemaker_session=sagemaker_session,
-            image_name=ecr_image,
-            framework_version=framework_version,
-            script_mode=True,
+            image_uri=ecr_image, framework_version=framework_version,
```

Comment:
Needs formatting fix.

Reply:
yes, this was made automatically by the command line tool. will fix formatting in final revision.